invenio_records_lom/config.py

```diff
@@ -15,6 +15,12 @@
 
 from .api import LomRecords
 
+
+def _(x):
```

Comment:
The underscore function is normally to translate or? What is that function then for?

Reply:
usually, a dummy placeholder until the translation is in place

Reply:
Would it be useful to add a warning, that only the identity function is used for translation? So no translation is missed?